test/ServerComparison.FunctionalTests/ResponseTests.cs

```diff
@@ -53,15 +53,15 @@ public Task ResponseFormats_Kestrel_ConnectionClose(ServerType serverType, Runti
             return ResponseFormats(serverType, runtimeFlavor, architecture, applicationBaseUrl, CheckConnectionCloseAsync);
         }
 
-        // [ConditionalTheory]
-        // [OSSkipCondition(OperatingSystems.Linux)]
-        // [OSSkipCondition(OperatingSystems.MacOSX)]
-        // // [InlineData(ServerType.IISExpress, RuntimeFlavor.Clr, RuntimeArchitecture.x64, "http://localhost:5078/")] // https://github.com/aspnet/IISIntegration/issues/7
```

Comment:
Does it mean that https://github.com/aspnet/IISIntegration/issues/7 can be closed now?